### ISSUE TITLE
Associated organisations mapping

### ISSUE THREAD

**Issue by badspyro** [UKHackspaceFoundation/website#87]
Since we are interested in working with other organisations that have a similar ethos or wish to lend a hand, such as university hack/makerspaces, we should have a similar mapping system for associate organisations as we do for members.

I suggest we have different colour pins for different types of organisations, on a separate map from the hack/makerspaces map to avoid confusion.

**Comment by amunizp** [comment]
I agree with adding them to a separate list as 'spaces that are not hackspaces'. But adding them to a map with a different colour might end up eclipsing the official ones. 'Men in sheds' could also ask to be listed and they are many.

If we are  doing it a separate map or a filter that is by default on could be better.

I would much rather add organisations that potentially could adjust to definition and not organizations that will never adjust to definition.
